process-engine/process_engine_contracts
# ✨ Add a State Flag for the Correlation

**Change notes taken from previous PR:** #99 

This had to be reverted, because the feature was not working properly.

**Changes:**

1.  Adds a StateFlag on the Correlation to determine, if the current Correlation is rather `active`, `finished` or error'ed out. 

**Issues:**

Related: https://github.com/process-engine/process_engine_runtime/issues/229

PR: #102

## How can others test the changes?

Try forcing a process instance into an error and see that the corresponding correlation entry is marked with an error.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
